configure/options.md

```diff
@@ -0,0 +1,58 @@
+Changing MySQL Options
+============================================================================
+
+MySQL allows the option to configure the database with a configuration file. You can pass the MySQL options from the [my.cnf](https://dev.mysql.com/doc/refman/8.0/en/option-files.html) configuration file to the cluster in one of the following ways:
+* CR.yaml
+* ConfigMap
+
+### Edit the CR.yaml
+
+Edit the configuration section of the deploy/cr.yaml. See the [PXC section]( https://percona.github.io/percona-xtradb-cluster-operator/configure/operator).
+
+```
+spec:
+  secretsName: my-cluster-secrets
+  pxc:
+    ...
+      configuration: |
+        [mysqld]
+        wsrep_debug=ON
+        [sst]
+        wsrep_debug=ON
+```
+
```

Comment:
See the [Custom Resource options, PXC section]( https://percona.github.io/percona-xtradb-cluster-operator/configure/operator) for more details.